### ISSUE TITLE
Decoding, no output in serial monitor

### ISSUE THREAD

**Issue by dakoeller** [sui77/rc-switch#329]
Dear All,

I am trying to decode the below 433mhz rf switch with ev1527, but without any luck. The serial monitor stays blank. I use a nodemcu  and a 5v rf receiver as a hardware. The nodemcu is powered via usb, the receiver is powered from the nodemcu vin pin. I used the receive_demo_advanced code and also tried the simple one but no success. Anybody have a clue what is the issue?

Thanks, Dave

![20200310_091244](https://user-images.githubusercontent.com/62008526/76292483-99f52480-62af-11ea-9579-ee6d61b5812b.jpg)
![20200310_091116](https://user-images.githubusercontent.com/62008526/76292484-9b265180-62af-11ea-84ec-76a82ddde6b6.jpg)

**Comment by NorthernMan54** [comment]
First comment your missing an Antenna, needs to have attached by the coil

Second comment, I’m in the same boat as your self but I’m running with a scope ( no software yet on the esp8266) attached to the output pins and see nothing.  I figure I have a dud board.

**Comment by NorthernMan54** [comment]
Also does your nodemcu supply 5 volts on the vin pin? I have one type of nodemcu that doesn’t.

**Comment by dakoeller** [comment]
Thank you for your comment! I tried now with a wire attached as an antenna but still nothing. Yes the board supplies 5v on the Vin I verified it with a multimeter. It is really annoying, it should work, the switch is supported by the rcswitch folder.

**Comment by dakoeller** [comment]
But often catches some random noise which is not the my switch:
![image](https://user-images.githubusercontent.com/62008526/76327791-b82c4600-62ea-11ea-9386-ce8ccb701ae1.png)

**Comment by 1technophile** [comment]
Hello, did you tried with another pin ?
Here is a list of the compatible ones:
https://docs.google.com/spreadsheets/d/1_5fQjAixzRtepkykmL-3uN3G5bLfQ0zMajM9OBZ1bx0/edit#gid=1864843263

**Comment by dakoeller** [comment]
Hello there! Thank you! I tried all the pins according to the link but none of it works. Still nothing happens in the serial monitor when i press the switch.

**Comment by 1technophile** [comment]
> I tried now with a wire attached as an antenna but still nothing

Does it measure 17.3 cm ?

> Hello there! Thank you! I tried all the pins according to the link but none of it works. Still nothing happens in the serial monitor when i press the switch.

Do you have another nodemcu with an emitter to load into it a senddemo and see if you have a compatibility issue with your switch?

**Comment by dakoeller** [comment]
Yes the antenna is ~173mm. Unfortunatelly I only have receivers, but I hoped it would work, because ev1527 is among the compatible devices. I also found several tutorials using the same hardwares.

**Comment by dakoeller** [comment]
Hi All! I made more testing, as 1technophile suggested I used an emitter to send codes via senddemo. This way there were no problem, the receiver put the codes to the serial monitor. So indeed the switch is the issue. I also found a schematic for it, and Ive found one interesting stuff: this one uses a ev1527 encoder chip, which has an output pin txd on the picture. But the output is not connected to anywhere. As for my understanding 433mhz transmitters should work the way they send the precoded key through the tx pin to the transmitter unit. Can someone explain how this switch works then??


![switch](https://user-images.githubusercontent.com/62008526/77623663-c2804e00-6f40-11ea-96a7-d2e9de964b5b.PNG)
